data/transactions/logic/eval.go

```diff
@@ -350,12 +434,17 @@ type EvalContext struct {
 	scratch scratchSpace
 
 	subtxns []transactions.SignedTxn // place to build for itxn_submit
-	// Previous transactions Performed() and their effects
-	InnerTxns []transactions.SignedTxnWithAD
-
-	cost    int // cost incurred so far
-	Logs    []string
```

Comment:
where did it go?

Reply:
cost remains, but InnerTxns and Logs are now kept in the EvalDelta that we are updating as we go anyway.